### DISCUSSION TITLE
1.3.0 업데이트 후 발생하는 문제

### DISCUSSION THREAD

**Discussion by kihye** [naverz/zepeto-studio-kor#767] [5. Q&A]
![image](https://user-images.githubusercontent.com/80503690/168714333-873fb06f-49ae-450a-94a0-25c2c5f4a717.png)

하위버전에서 업데이트 후에 발생하는 문제같습니다.
이것저것 만져봐도 멀티플레이 코드가 나오질 않네요.
스크린샷 예제 프로젝트와 비교해가면서 차이점 찾아보는 중인데 뭔가 특별한 차이라던가 그런건 안보입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요.  ASSETS > Reimport All 을 하신 후 다시 찾아보시기를 추천 드립니다.

**Reply by kihye** [reply]
감사합니다. 이 방법을 적용하니까 다시 잘 나옵니다.
